### ISSUE TITLE
Endpoints need to be updated

### ISSUE THREAD

**Issue by Alex313031** [ltguillaume/thorium-winupdater#4]
@ltguillaume  Since I am always releasing AVX2 and SSE3 versions now, I decided to just release them all together in the main Thorium-Win repo. As such, we need to update the endpoints. See the latest release notes in that repo.

**Comment by ltguillaume** [comment]
I've downloaded the new releases and currently, the `thor_ver` files are inconsistent, so let's fix that:

1. The first line of the `thor_ver` file should contain the part of the repo name after `Thorium-`, so `Win` here, `Win7` for the other releases for Windows 7.
2. The second line of the `thor_ver` file should contain (only) the part of the filename after `thorium_` and before `_120.0.6099.235*.*`, so `AVX2`, `AVX`, `SSE3`, `SSE4`, `WIN32_SSE2` or `WIN32_SSE3`.
3. The tag name should contain the actual version number, ideally prepended with `M` (so in this case `M119.0.6045.214`, notice the `.0.`), just like in the other Thorium repos.
4. https://github.com/Alex313031/thorium-win7/releases has a _separate_ release for the 32-bit files of the same version. That introduces a whole new problem for an updater. They should all be bundled.

**Comment by Alex313031** [comment]
@ltguillaume Does this look good?

https://github.com/Alex313031/thorium/commit/bba665e188d0292e879bae5e7205950ff73f2813
&
https://github.com/Alex313031/thorium-win7/commit/daf8839a6af77ba0eb62c432a82bb1d2230cffd2

**Comment by ltguillaume** [comment]
Yes, looks great. I'll update WinUpdater as soon as these new thor_ver files make it to the releases.

**Comment by alsdfjldafsjalsfjk** [comment]
Hi, i think I'm getting a error related to this issue, every time i start my browser the updater checks and fails to find something: `Could not find the URL to download Thorium. No changes were made to your Thorium folder.` This also happens during scheduled and also on manual updates.

My version of the updater is: 1.8.1 and Thorium is 12.0.0.6099.235

**Comment by ltguillaume** [comment]
@alsdfjldafsjalsfjk You'll really need to update manually for now. Until the new Thorium releases are out with properly formed `thor_ver` files _and_ properly formed tag names for the GitHub releases, there's no way I can update WinUpdater to work consistently for updates.

**Comment by darkerraven** [comment]
> @alsdfjldafsjalsfjk You'll really need to update manually for now. Until the new Thorium releases are out with properly formed `thor_ver` files _and_ properly formed tag names for the GitHub releases, there's no way I can update WinUpdater to work consistently for updates.

now the new update of Thorium released so will you update WinUpdater?

**Comment by alsdfjldafsjalsfjk** [comment]
> @alsdfjldafsjalsfjk You'll really need to update manually for now. Until the new Thorium releases are out with properly formed `thor_ver` files _and_ properly formed tag names for the GitHub releases, there's no way I can update WinUpdater to work consistently for updates.

Thanks! Just updated and ran winupdater after and its not giving a url error anymore! Thanks again!

**Comment by ltguillaume** [comment]
> > @alsdfjldafsjalsfjk You'll really need to update manually for now. Until the new Thorium releases are out with properly formed `thor_ver` files _and_ properly formed tag names for the GitHub releases, there's no way I can update WinUpdater to work consistently for updates.
> 
> now the new update of Thorium released so will you update WinUpdater?

@darkerraven I just released v1.8.3 (WinUpdater will automatically update itself), which should update the versions from https://github.com/Alex313031/Thorium-Win/releases/ properly (it also includes some logic to do that from older versions that still have inconsistent `thor_ver` files). The builds in https://github.com/Alex313031/Thorium-Win7/releases/ are still impossible to properly update, because not only are the `thor_ver` files for x64 and x86 builds exactly the same, but the releases of these architectures have been separated, even though they're for the same Chromium version (see https://github.com/ltguillaume/thorium-winupdater/issues/4#issuecomment-1908146475).
